gprofiler/merge.py

```diff
@@ -257,6 +258,7 @@ def _make_profile_metadata(
         "metrics": metrics.__dict__,
         "application_metadata": application_metadata,
         "application_metadata_enabled": application_metadata_enabled,
+        "profiling_mode": profiling_mode,
```

Comment:
You can extract it from `meetadata` here so it's an explicit field

Reply:
No need to pass an additional field down the chain

Reply:
done